### PR TITLE
v0.5.0 to staging

### PR DESCRIPTION
**Pull request by stolarczyk**
wait for: 

- [x] [`refgenconf`](https://github.com/refgenie/refgenconf/pull/95)

to do:

- [x] bump `refgenconf` requirement
- [x] bump `ubiquerg` requirement

### CODE REVIEW

**Comment by nsheff** [comment]
should we set up the new idea for deploying these containers via actions as we do this release?

**Comment by stolarczyk** [comment]
yes, I think we should

**Comment by nsheff** [comment]
Ok I'll look into it later today -- we should set that up before merging into staging, I guess

**Comment by nsheff** [comment]
ok I've moved it into the new org, and everything is set up and running correctly...

File: refgenieserver/helpers.py [hunk 1]
```diff
@@ -103,3 +105,31 @@ def get_openapi_version(app):
...
+    either a remote URL to the file or a file path along with a flag indicating
+    the source
+
+    :param refgenconf.RefGenConf rgc: configiration object to use
+    :param dict fill_dict: a dictionary to use to fill in the path template
+    :param str fill_dict: the path template
+    :return (str, bool): a pair of file source and the flag indicationg whether
+     the source is remote
+    """
+    CFG_REMOTE_URL_BASE_KEY = "remote_url_base"
```
Comment (stolarczyk) [inline refgenieserver/helpers.py]: import from `refgenconf` once released

**Comment by nsheff** [comment]
let's bump this to 0.5.0 since it adds the new s3 capability

**Pull request merged**